store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/util/ConcatenateMergeFunctionTest.java

```diff
@@ -78,11 +79,42 @@ public void shouldReturnUpdateWrappedInIterableWhenStateNull() {
 
         // Then
         assertThat(results)
+                .hasSameClassAs(results)
                 .containsExactly(update);
     }
 
     @Test
-    public void shouldConcatenateTwoIterablesIntoOne() {
+    public void shouldReturnIntegerUpdateWrappedInIterableWhenStateNull() {
+        // Given
+        final ConcatenateMergeFunction mergeFunction = new ConcatenateMergeFunction();
+
+        // When
+        Object update = 25;
+        Iterable<Object> results = mergeFunction.apply(update, null);
+
+        // Then
+        assertThat(results)
```

Comment:
`assertThat(results).hasSameClassAs(results)` - won't that always pass?

Reply:
You're right, it will. I was using this as a lazy way of doing `assertThat(results).isInstanceOf(Iterable.class)`, but as we're assigning to an object of that class, if it was anything else we'd have an exception thrown prior to the test.

I'll remove these asserts.